### PR TITLE
feat: make use of SI/binary prefixes consistent for disk, network, and memory

### LINKED ISSUE #1631: Make use of SI/binary prefixes consistent for disk, network, and memory
### Main idea

Please refer to https://github.com/lablup/giftbox/issues/186

### Alternative ideas

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by lizable**
This PR resolves #1631.

### CODE REVIEW

File: src/components/backend-ai-session-list.ts [hunk 1]
```diff
@@ -931,13 +919,17 @@ export default class BackendAiSessionList extends BackendAIPage {
...
-    return Number(value / 1000).toFixed(0);
+  /**
+   * Convert the value bytes to MB with decimal point to 1 as a default
+   *
+   * @param {number} value
+   * @param {number} decimalPoint decimal point to show
+   * @return {string} converted value from Bytes to MB
+   */
+  static bytesToMB(value, decimalPoint = 1) {
+    return Number(value / (10 * 9)).toFixed(1);
```
Comment (agatha197) [inline src/components/backend-ai-session-list.ts]:
```suggestion
    return Number(value / (10 ** 6)).toFixed(1);
```

File: src/components/backend-ai-session-view.ts [hunk 2]
```diff
@@ -322,12 +322,15 @@ export default class BackendAiSessionView extends BackendAIPage {
...
-    return Number(value / (1024 * 1024)).toFixed(1);
+  /**
+   * Convert the value bytes to MiB with decimal point to 1 as a default
+   *
+   * @param {number} value
+   * @param {number} decimalPoint decimal point to show
+   * @return {string} converted value from Bytes to MiB
+   */
+  static bytesToMiB(value, decimalPoint = 1) {
+    return Number(value / (2 * 20)).toFixed(1);
```
Comment (agatha197) [inline src/components/backend-ai-session-view.ts]:
```suggestion
    return Number(value / (2 ** 20)).toFixed(1);
```

File: src/components/backend-ai-storage-proxy-list.ts [hunk 3]
```diff
@@ -511,13 +498,13 @@ export default class BackendAIStorageProxyList extends BackendAIPage {
...
-                                     description="${this.storageProxyDetail.current_mem}GB/${this.storageProxyDetail.mem_slots}GB"
+                                     description="${this.storageProxyDetail.current_mem} GiB / ${this.storageProxyDetail.mem_slots} GiB"
                 ></lablup-progress-bar>
               </div>
               <h3>Network</h3>
               ${'live_stat' in this.storageProxyDetail && 'node' in this.storageProxyDetail.live_stat ? html`
-                <div>TX: ${this._bytesToMB(this.storageProxyDetail.live_stat.node.net_tx.current)}MB</div>
-                <div>RX: ${this._bytesToMB(this.storageProxyDetail.live_stat.node.net_rx.current)}MB</div>
+                <div>TX: ${BackendAIStorageProxyList.bytesToMB(this.storageProxyDetail.live_stat.node.net_tx.current)}MB</div>
+                <div>RX: ${BackendAIStorageProxyList.bytesToMB(this.storageProxyDetail.live_stat.node.net_rx.current)}MB</div>
```
Comment (agatha197) [inline src/components/backend-ai-storage-proxy-list.ts]:
```suggestion
                <div>TX: ${BackendAIStorageProxyList.bytesToMB(this.storageProxyDetail.live_stat.node.net_tx.current)} MB</div>
                <div>RX: ${BackendAIStorageProxyList.bytesToMB(this.storageProxyDetail.live_stat.node.net_rx.current)} MB</div>
```

**Review by Sujin-Kim1 — CHANGES_REQUESTED** [review]
I found some bugs. But it looks good except for them.

**Review by Sujin-Kim1 — APPROVED** [review]
LGTM

**Pull request merged**